.gitignore

```diff
@@ -21,3 +21,6 @@ bazel-*
 packages.bzl
 BUILD
 WORKSPACE
+
+# Deps of checked-in dart2js generated code
+*.dart.js.deps
```

Comment:
Lest be a bit more specific here, also see https://github.com/dart-lang/build/pull/1734 for the custom build script which we can update to also run on this new entrypoint.